actions/image_partition_action.go

```diff
@@ -58,6 +58,7 @@ a 32 bits hexadecimal number (e.g. '1234ABCD' without any dash separator).
 	   fsck: bool
 	   fsuuid: string
 	   partuuid: string
+	   partattrs: list of partition attributes
```

Comment:
```suggestion
	   partattrs: list of partition attributes bits to set
```
Would be good to have the `Verify` function check it's only bit numbers in a valid range

Reply:
Sorry for the delay. I've implemented verification now.

However I've noticed that bits 0-2 despite being documented in the specification and in the sfdisk man page, don't seem to be supported:

```
2024/05/31 19:42:27 sfdisk | unsupported GPT attribute bit '2'
2024/05/31 19:42:27 sfdisk | sfdisk: /dev/disk/by-fakemachine-label/fakedisk-0: partition 1: failed to set partition attributes
```

Should we forbid these values in verification too? Given it's documented it might be something specific to my setup. Unfortunately I don't have a lot of time to look into this.

Reply:
However, it's possible to set them using their name i.e. `LegacyBIOSBootable` instead of `2` in this case.